providers/shared/components/queuehost/id.ftl

```diff
@@ -0,0 +1,153 @@
+[#ftl]
+
+[@addComponentDeployment
+    type=QUEUEHOST_COMPONENT_TYPE
+    defaultGroup="solution"
+/]
+
+[@addComponent
+    type=QUEUEHOST_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type" : "Description",
+                "Value" : "Managed message queue hosting"
+            }
+        ]
+    attributes=
+        [
+            {
+                "Names" : "Engine",
+                "Type" : STRING_TYPE,
+                "Values" : [ "rabbitmq" ],
```

Comment:
Should this value be added by the provider implementing it?

Reply:
rabbitmq would be a standard implementation of a queue host and would be supported across multiple providers. So we could do it per provider but we don’t currently do that for dB and cache which have a similar configuration

Reply:
Agree with @roleyfoley - if a standalone product like rabbit, its reasonable to have a standard value in engine